SLF4J: The requested version 1.5.8 by your slf4j binding is not compatible with [1.6, 1.7]
SLF4J: See http://www.slf4j.org/codes.html#version_mismatch for further details.
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
java.lang.ArrayIndexOutOfBoundsException: -1
	at game.content.shop.ShopAssistant.reduceQuantityFromShop(ShopAssistant.java:435)
	at game.content.shop.ShopAssistant.buyItemAction(ShopAssistant.java:470)
	at game.content.shop.ShopAssistant.buyFromPointsShop(ShopAssistant.java:64)
	at game.content.shop.ShopAssistant.buyItem(ShopAssistant.java:691)
	at game.content.packet.Bank5Packet.processPacket(Bank5Packet.java:61)
	at network.packet.PacketHandler.processPacket(PacketHandler.java:373)
	at game.player.Player.processQueuedPackets(Player.java:6578)
	at game.player.PlayerHandler.packetProcessing(PlayerHandler.java:187)
	at core.Server$1.lambda$1(Server.java:368)
	at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:407)
	at core.Server$1.execute(Server.java:368)
	at core.maintick.Task.tick(Task.java:111)
	at core.maintick.TaskScheduler.run(TaskScheduler.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Server knows this nigga got a pet summoned.
Server knows this nigga got a pet summoned.
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
Server knows this nigga got a pet summoned.
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
Traceback (innermost last):
  File "<iostream>", line 29, in quest_button_2
NameError: quest_name
java.lang.ArrayIndexOutOfBoundsException: -1
	at game.content.shop.ShopAssistant.reduceQuantityFromShop(ShopAssistant.java:435)
	at game.content.shop.ShopAssistant.buyItemAction(ShopAssistant.java:470)
	at game.content.shop.ShopAssistant.buyFromPointsShop(ShopAssistant.java:64)
	at game.content.shop.ShopAssistant.buyItem(ShopAssistant.java:691)
	at game.content.packet.BankAllPacket.processPacket(BankAllPacket.java:55)
	at network.packet.PacketHandler.processPacket(PacketHandler.java:373)
	at game.player.Player.processQueuedPackets(Player.java:6578)
	at game.player.PlayerHandler.packetProcessing(PlayerHandler.java:187)
	at core.Server$1.lambda$1(Server.java:368)
	at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:407)
	at core.Server$1.execute(Server.java:368)
	at core.maintick.Task.tick(Task.java:111)
	at core.maintick.TaskScheduler.run(TaskScheduler.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
